spacy/gold/example.pyx

```diff
@@ -26,6 +27,17 @@ cpdef Doc annotations2doc(vocab, tok_annot, doc_annot):
     return output
 
 
+class SavedAlignment:
+    def __init__(self,
+        alignment: Optional[Alignment] = None,
+        predicted_tokens: Optional[List[str]] = None,
+        reference_tokens: Optional[List[str]] = None,
+    ):
```

Comment:
I personally wouldn't have the default values in this constructor, and only create this object if all 3 parameters are available (and have the `self._saved_alignment` field be `None` otherwise)